src/gt4py/next/otf/languages.py

```diff
@@ -57,6 +57,11 @@ class Python(LanguageTag):
     ...
 
 
+class JSON(LanguageTag):
```

Comment:
These type tags are about making it clear which steps can be mixed with which ones. Therefore it is important to describe what semantics are used. For your purposes "SDGF" describes the semantics, "JSON" only the format.

```suggestion
class SDFG_JSON(LanguageTag):
```

Reply:
I see, it makes sense. I will use "SDFG" in this case.